bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/builder/ChannelBuilder.java

```diff
@@ -87,7 +90,7 @@ private ChannelBuilder(ChannelUID channelUID, String acceptedItemType, Set<Strin
      * @param properties properties to add
      * @return channel builder
      */
-    public @NonNull ChannelBuilder withProperties(Map<@NonNull String, String> properties) {
+    public ChannelBuilder withProperties(Map<@NonNull String, String> properties) {
```

Comment:
The `@NonNull` could be removed for the map's key.

Reply:
Oh sorry I overlooked that one, I have removed it now.